parsers/flutter.go

```diff
@@ -0,0 +1,79 @@
+package parsers
+
+import (
+	"encoding/json"
+	"github.com/cirruslabs/cirrus-ci-annotations/model"
+	"io"
+	"os"
+	"strings"
+)
+
+type flutterTest struct {
+	ID     int64
+	Name   string
+	URL    string
+	Line   int64
+	Column int64
+}
+
+type flutterEntry struct {
+	Type   string
+	TestID int64
+	Result string
+	Test   flutterTest
+}
+
+func annotationFromTest(test flutterTest) model.Annotation {
+	return model.Annotation{
+		Type:    model.TestResultAnnotationType,
+		Level:   "warning",
```

Comment:
Should it be error here?

Reply:
Changed in https://github.com/cirruslabs/cirrus-ci-annotations/pull/29/commits/9a36da8fca1b908f39536cc027d5eca1910600f5.